dev/build/makepack-howto.md

```diff
@@ -0,0 +1,142 @@
+# Dolibarr Makepack How To
+
+This documentation describe steps to build a BETA or RELEASE versions of Dolibarr.
+There is a chapter for BETA version and a chapter for RELEASE version.
+
+
+## Prerequisites on Linux
+
+Prerequisites to build the tgz, debian and rpm packages:
+
+`apt-get install perl tar dpkg dpatch p7zip-full rpm zip php-cli`
+
+Prerequisites to build autoexe DoliWamp package from Linux (solution seems broken since Ubuntu 20.04+):
+
+`apt-get install wine q4wine`
+
+* Launch "wine cmd" to check a drive Z: pointing to / exists.
+* Install InnoSetup
+
+  For example by running isetup-5.5.8.exe (https://www.jrsoftware.org)  https://files.jrsoftware.org/is/5/
+
+* Install WampServer into "C:\wamp64" to have Apache, PHP and MariaDB
+
+  For example by running wampserver3.2.6_x64.exe (https://www.wampserver.com).
+See file dev/build/exe/doliwamp.iss to know the doliwamp version currently setup.
+
+* Add path to ISCC into PATH windows var:
+
+  Launch wine cmd, then regedit and add entry int HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment\PATH
+
+* To build manually the .exe from Windows
+
+  Note: running from makepack-dolibarr.pl script is however recommended
+  open file dev/build/exe/doliwamp.iss and click on button "Compile".
+  The .exe file will be build into directory build.
+
+
+## Prerequisites on Windows
+
+Prerequisites to build autoexe DoliWamp package from Windows:
+
+* Install Perl for Windows (https://strawberryperl.com/)
+* Install isetup-5.5.8.exe (https://www.jrsoftware.org)
+* Install Microsoft Visual C++ Redistributable 2017 (https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170)
+* Install WampServer-3.2.6-64.exe (Apache 2.4.51, PHP 7.4.26, MariaDB 10.6.5 for example. Version must match the values found into doliwamp.iss)
+* Install GIT for Windows (https://git-scm.com/ => You must choose option "Add Git bash profile", "Git commit as-is")
+* Install Dolibarr current version:
+  `git clone https://github.com/dolibarr/dolibarr  or  git clone --branch X.Y https://github.com/dolibarr/dolibarr`
+
+* Add the path of PHP (C:\wamp64\bin\php\php7.4.26) and InnoSetup (C:\Program Files (x86)\Inno Setup 5) into the %PATH% of Windows.
+
+* Create a config file c:\dolibarr\dolibarr\htdocs\conf\conf.php with content
+
+```
+  <?php
+  $dolibarr_main_document_root="c:\dolibarr\dolibarr\htdocs";
+  $dolibarr_main_url_root='http://localhost';
+```
+
+
+## Actions to do a BETA
+
+This files describe steps made by Dolibarr packaging team to make a beta version of Dolibarr, step by step.
+
+* Check all files are committed.
+* Update version/info in ChangeLog, for this you can:
+
+To generate a changelog of a **major new version** x.y.0 (from a repo on branch develop), you can do
+
+```
```

Comment:
You can use `bat` after the ticks to indicate windows commands, or `bash` for unix.